src/main/java/org/opentripplanner/middleware/utils/DateTimeUtils.java

```diff
@@ -153,6 +154,13 @@ public static ZoneId getOtpZoneId() {
         return ZoneId.of(otpTzId);
     }
 
+    /**
+     * Converts a {@link LocalDateTime} on OTP's time zone to epoch milliseconds.
```

Comment:
on -> in

Reply:
fixed in https://github.com/ibi-group/otp-middleware/pull/102/commits/911c618356112c4db4b089e158aab92ab6c23062.